src/setup/extract-configuration.go

```diff
@@ -60,6 +61,8 @@ type SubExperiment struct {
 	FunctionImageSizeMB     float64  `json:"FunctionImageSizeMB"`
 	DataTransferChainLength int      `json:"DataTransferChainLength"`
 	StorageTransfer         bool     `json:"StorageTransfer"`
+	Handler                 string   `json:"Handler"`
```

Comment:
Could you add these field to the experiment JSON files as well, please? At least the ones running hellopy functions?